EIPS/EIP-X-on-chain-issuance-control.md

```diff
@@ -0,0 +1,119 @@
+---
+eip: <to be assigned>
```

Comment:
Please number this 1015 and rename to eip-1015.md.

Reply:
Can I use eip 1000, since it's currently unclaimed 😉